ngraph/frontend/frontend_manager/include/frontend_manager/frontend_manager.hpp

```diff
@@ -96,11 +96,10 @@ namespace ngraph
     } // namespace frontend
 
     template <>
-    class FRONTEND_API VariantWrapper<std::shared_ptr<std::istream>>
-        : public VariantImpl<std::shared_ptr<std::istream>>
+    class FRONTEND_API VariantWrapper<std::istream*> : public VariantImpl<std::istream*>
     {
     public:
-        static constexpr VariantTypeInfo type_info{"Variant::std::shared_ptr<std::istream>", 0};
+        static constexpr VariantTypeInfo type_info{"Variant::std::istream*", 0};
```

Comment:
What happens in this situation:

```
std::make_shared<VariantWrapper<std::istream*>> variant;
{
    std::istream stream;
    variant = make_shared<VariantWrapper<std::istream*>>(&stream);
}
```
Should variant control the lifetime of istream?

Reply:
No, this situation should be avoided. User controls the lifetime of a stream, not variant or frontend.